migration/iD12/oracle/202406181038_IDEMPIERE-6177.sql

```diff
@@ -0,0 +1,81 @@
+;
+
+-- Jun 18, 2024, 10:44:03 AM CEST
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,XPosition,ColumnSpan) VALUES (208492,'Generate Invoice','Automatically Generate Invoice after (Sales/Purchase) Order completed','When selected the Order will generate Invoice automatically. Note that only Orders with document subtype Standard and Prepay Order can be applied!',167,216612,'Y',1,370,'N','N','N','N',0,0,'Y',TO_TIMESTAMP('2024-06-18 10:44:02','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-06-18 10:44:02','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D','78a968cd-135c-42a3-9da7-a75057cc703d','Y',330,2,2)
+;
+
+-- Jun 18, 2024, 10:44:03 AM CEST
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,XPosition,ColumnSpan) VALUES (208493,'Generate Shipment','Automatically Generate Shipment after (Sales/Purchase) Order completed','When selected the Order will generate Shipment automatically. Note that only Orders with document subtype Standard and Prepay Order can be applied!',167,216613,'Y',1,380,'N','N','N','N',0,0,'Y',TO_TIMESTAMP('2024-06-18 10:44:03','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-06-18 10:44:03','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D','fbb23049-e37b-40ad-b414-93b789dff9ed','Y',340,2,2)
+;
+
+-- Jun 18, 2024, 10:46:25 AM CEST
+UPDATE AD_Field SET SeqNo=330,Updated=TO_TIMESTAMP('2024-06-18 10:46:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=6567
+;
+
+-- Jun 18, 2024, 10:46:25 AM CEST
+UPDATE AD_Field SET SeqNo=340,Updated=TO_TIMESTAMP('2024-06-18 10:46:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=3125
+;
+
+-- Jun 18, 2024, 10:46:25 AM CEST
+UPDATE AD_Field SET DisplayLogic='@DocBaseType@=''SOO'' | @DocBaseType@=''POO''', SeqNo=350,Updated=TO_TIMESTAMP('2024-06-18 10:46:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=208492
+;
+
+-- Jun 18, 2024, 10:46:25 AM CEST
+UPDATE AD_Field SET DisplayLogic='@DocBaseType@=''SOO'' | @DocBaseType@=''POO''', SeqNo=360,Updated=TO_TIMESTAMP('2024-06-18 10:46:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=208493
+;
+
+-- Jun 18, 2024, 10:46:25 AM CEST
+UPDATE AD_Field SET SeqNo=0,Updated=TO_TIMESTAMP('2024-06-18 10:46:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=204719
+;
+
+-- Jun 18, 2024, 10:56:01 AM CEST
+UPDATE AD_Field SET DisplayLogic='@OrderType@=''SO'' | 
+@OrderType@=''PR''', SeqNo=90,Updated=TO_TIMESTAMP('2024-06-18 10:56:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=1094
```

Comment:
why the new line at lines 60, 69 and 78?

oracle have problems with lines starting with @ 

please fix this in both oracle and postgresql scripts

Reply:
Done